### DISCUSSION TITLE
Epic games version of the game isnt updated to the latest version

### DISCUSSION THREAD

**Discussion by microwave805** [FakeFishGames/Barotrauma#15136] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The epic games version of the game is at 1.6.18.1, while on steam its 1.6.19.1 thus making you not be able to play with people playing on the steam version of the game

Expected behaviour is the game being on 1.6.19.1, and being able to play with steam players on the latest version.

### Reproduction steps

Press the 3 dots in the epic games launcher
Then version should pop up. Observe that the version is 1.6.18.1
Alternatively, Launch the game from the epic games launcher and observe that you cant join any steam players in the latest version of the game

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Other

### -

Game version issue.

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report!

We are aware of the issue - unfortunately we're currently unable to publish the latest build due to an issue in the Epic Store's backend that we cannot resolve ourselves, and are waiting for support from Epic to get this sorted out. I'm sorry for the inconvenience!
